helm/syntho-ui/values.yaml.tpl

```diff
@@ -20,7 +20,7 @@ core:
   volumes: []
   volumeMounts: []
   database_enabled: true
-  celery_args: ['-A', 'app.celery', 'worker', '--loglevel=info', '--concurrency=1', '--max-memory-per-child=50000']
+  celery_args: ['-A', 'app.celery', 'worker', '--loglevel=info', '--concurrency=4', '--max-memory-per-child=50000']
```

Comment:
I think the worker would struggle with 4 concurrency with [this resource config](https://github.com/syntho-ai/deployment-tools/blob/20abe0128ab154b4635819316b635f131298083e/helm/syntho-ui/templates/core/core-worker-pod.yaml#L85-L93). I think if we want to scale it, then it would be better if it'd scale it horizontally as pod. But it is a whole different concept and actually I was thinking to bring horizontal pod scaling to the table. But we can talk about it later for now. But at least resource config in celery worker should at least have 4 CPU as a `limit` for this to work well.

Reply:
This one I'll put back to one after a particular fix is done on the Syntho stack side. There is basically a celery task there that might cause the others to freeze up because it's sometimes running for hours. Somebody is currently working on a fix there. If that's fixed, I agree this can be set to 1 to match the amount of CPUs reserved for the pod.